sqldelight-runtime/src/main/java/com/squareup/sqldelight/ColumnAdapter.java

```diff
@@ -15,17 +15,17 @@
  */
 package com.squareup.sqldelight;
 
-import android.content.ContentValues;
-import android.database.Cursor;
-
-/** Marshal and map the type {@code T} to and from a representation in the database. */
-public interface ColumnAdapter<T> {
+/**
+ * Marshal and map the type {@code T} to and from a database type {@code S} which is one of Long,
+ * Double, String, byte[]
+ */
+public interface ColumnAdapter<T, S> {
   /**
    * Return an instance of {@code T} corresponding to the value at {@code columnIndex} in
    * {@code cursor}.
```

Comment:
i got rid of them because i feel like my comment would be "encode S as T" or something lol